cli/root.go

```diff
@@ -632,52 +622,58 @@ func (r *RootCmd) createAgentClient() (*agentsdk.Client, error) {
 	return client, nil
 }
 
-// CurrentOrganization returns the currently active organization for the authenticated user.
-func CurrentOrganization(r *RootCmd, inv *serpent.Invocation, client *codersdk.Client) (codersdk.Organization, error) {
-	conf := r.createConfig()
-	selected := r.organizationSelect
-	if selected == "" && conf.Organization().Exists() {
-		org, err := conf.Organization().Read()
-		if err != nil {
-			return codersdk.Organization{}, xerrors.Errorf("read selected organization from config file %q: %w", conf.Organization(), err)
-		}
-		selected = org
-	}
+type OrganizationContext struct {
+	// FlagSelect is the value passed in via the --org flag
+	FlagSelect string
+}
 
-	// Verify the org exists and the user is a member
+func NewOrganizationContext() *OrganizationContext {
+	return &OrganizationContext{}
+}
+
+func (o *OrganizationContext) AttachOptions(cmd *serpent.Command) {
+	cmd.Options = append(cmd.Options, serpent.Option{
+		Name:        "Organization",
+		Description: "Select which organization (uuid or name) to use.",
+		// Only required if the user is a part of more than 1 organization.
+		// Otherwise, we can assume a default value.
+		Required:      false,
+		Flag:          "org",
+		FlagShorthand: "z",
```

Comment:
`-z` is a really weird choice here imo. I know `-o` is used, but what about `-O`? just doesn't feel like there is very clear connection.

Reply:
I was thinking organiZation lol. I can switch to `-O`